features/cloud/info.yaml

```diff
@@ -3,6 +3,7 @@ type: element
 features:
   include:
     - server
+    - _secureboot
```

Comment:
secureboot should not be enabled by default as it removes legacy boot support and prevents loading user build kernel modules, or at least makes this significantly more challenging (see #876)
&xrArr; should not be included from cloud feature

Reply:
Ok, removed it.